### PR TITLE
(CU-861m74hnp) add how to lp guide to book

### PR DESCRIPTION
**Pull request by Martin-Kiss**

### CODE REVIEW

**Review by 0xBrainjar2 — APPROVED** [review]

File: docs/docs/user-guides/how-to-provide-liquidity.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+![pool_page](./images-how-to-provide-liquidity/pool-page.png)
+
+1. On [app.pablo.finance], connect your wallet:
+2. Navigate to the "Pool" page
+
+[app.pablo.finance]: https://app.pablo.finance
+
+3. Choose one of the availabe pools, currently:
+
+- KSM/USDT
```
Comment (benluelo) [inline docs/docs/user-guides/how-to-provide-liquidity.md:31]: Does this need to be repeated?

**Review by benluelo — APPROVED** [review]

**Comment by Martin-Kiss** [comment]
@Mergifyio queue

**Comment by Martin-Kiss** [comment]
@Mergifyio update

**Comment by Martin-Kiss** [comment]
@Mergifyio queue

**Comment by Martin-Kiss** [comment]
@Mergifyio update

**Comment by Martin-Kiss** [comment]
@Mergifyio queue

**Pull request merged**